frontend/views/pages/Contributions.vue

```diff
@@ -2,6 +2,13 @@
 page(pageTestName='contributionsPage' pageTestHeaderName='contributionsTitle')
   template(#title='') {{ L('Contributions') }}
 
+  page-section.c-card-empty(v-if='true')
```

Comment:
Why `v-if='true'`? Can you use a real value here?

Reply:
for now let's keep it this way until we have contributions page fully implemented with real data, otherwise this will disappear the first time we add our details